redbot/core/commands/commands.py

```diff
@@ -57,6 +58,49 @@ def __init__(self, *args, **kwargs):
             checks=getattr(decorated, "__requires_checks__", []),
         )
 
+    def format_help_for_context(self, ctx: "Context") -> str:
```

Comment:
It might also be good to mention here that this method can (is intended to) be subclassed.

Reply:
It's mentioned in the docstring already.